app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+
+# ページごとに完全なタイトルを返す
+    def full_title(page_title = "")
+        base_title = "Photo Spots"
+        if page_title.empty?
+            base_title
+         else
+            page_title + "|" + base_title
+        end
+    end
```

Comment:
MUST
このままの形で行くなら、こんな感じで直して欲しいです！👇
- コメントアウトをメソッドと同じインデントに揃える
- 基本的に、文字列展開(ex: `"#{page_title}"`)など、ダブルクオーテーションじゃないといけない時以外はシングルクオーテーションを使うことが良いとされているので、直して欲しいです！(`""`→`''`)
- `else`のインデントが`if 〜 end`と揃ってないので揃えて欲しい
```suggestion
    # ページごとに完全なタイトルを返す
    def full_title(page_title = '')
        base_title = 'Photo Spots'
        if page_title.empty?
            base_title
        else
            page_title + '|' + base_title
        end
    end
```

Reply:
IMO
ちなみに私なら、こんな感じで書きます！👇
三項演算子は複雑なif文(nestするifとか)には向いてませんが、簡単なのだと三項演算子で書くことが多いですね 🙆 (Ref: [【Ruby・Rails】三項演算子(条件演算子）を使ってif文をスリムに書こう！ - Qiita](https://qiita.com/lasershow/items/160c854e4256ba596ec5))
```ruby
    # ページごとに完全なタイトルを返す
    def full_title(page_title = '')
        base_title = 'Photo Spots'
        page_title.empty? ? base_title : "#{page_title} | #{base_title}"
    end
```

Reply:
レビューありがとうございます！！
三項演算子！これを機に覚えたいので、MUSTもIMOもどちらも修正しました☺️！